tests/AutoReview/ProjectCodeTest.php

```diff
@@ -452,6 +452,56 @@ public function testExpectedInputOrder(string $testClassName): void
         }
```

Comment:
> I am wondering if it wouldn't be possible to introduce a fixed contract for fixers' test classes

I think the pre-step for it is unification of prototypes (eg in this PR we see `config, expected, input` -> `expected, input, config` (yet not claiming this PR unifies everything) and test method names (especially around `@requires PHP xyz`)